test/docker/docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
 
   tigris_search:
     container_name: tigris_search
-    image: typesense/typesense:0.23.1
+    image: tigrisdata/typesense:0.24.0
```

Comment:
We can use the upstream image here directly. If we don't we have to change the location of the binary from /usr/bin/typesense to /opt/typesense or sth along those lines. You can find the right location in `go/infra`.